acid/templates/status.html

```diff
@@ -1,6 +1,7 @@
 {% extends "base.html" %}
 {% block title %}Status{% endblock %}
 {% block content %}
+    <meta id="refs_list" content="{{refs_list}}">
```

Comment:
Meta tags **must always** be in `<head></head>` section.
You can create new, empty block in `base.html` and then extend it like you do with `block scripts`.